src/components/common/Switch.tsx

```
@@ -0,0 +1,22 @@
+interface SwitchProps {
+    label: string;
+    description?: string;
+    onClick: () => void;
+    value: number
+  }
+
+const Switch = ({label, description, value, onClick}:SwitchProps) => {
+  return (
+    <div onClick={onClick} className="flex gap-4 lg:flex-row flex-col lg:items-center cursor-pointer">
+      <Typography className=" text-sm" fw={900}>{label}</Typography>
+    <input key={value} defaultChecked={value === 0 ? false: true } type="checkbox" value={value} className="sr-only peer"/>
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)